app/src/main/res/values/strings.xml

```diff
@@ -290,4 +290,7 @@
         <item quantity="one"><xliff:g>%1$d</xliff:g> card (<xliff:g id="archivedCount">%2$d</xliff:g> archived)</item>
         <item quantity="other"><xliff:g>%1$d</xliff:g> cards (<xliff:g id="archivedCount">%2$d</xliff:g> archived)</item>
     </plurals>
-</resources>
\ No newline at end of file
+    <string name="previousCard">Previous card</string>
+    <string name="nextCard">Next card</string>
+    <string name="cardNoAdditionalInfo">This card has no additional data. When you add a note, add it to a group, set balance or set expiry it will appear here.</string>
```

Comment:
```suggestion
    <string name="cardNoAdditionalInfo">You can add a note, assign a group, set a balance or expiry.</string>
```

Reply:
This doesn't make sense to me, because you can't do any of those things in the context where the message is shown.